packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/OutputEditor.js

```diff
@@ -33,12 +39,24 @@ export default class OutputEditor extends Component {
     }
   }
 
+  renderValidation() {
+    if (this.isNameInvalid) {
+      return (
+        <div className='output-warning' style={ { color: 'red' } }>
+          Whitespaces not allowed
+        </div>
+      );
+    }
+    return (<div><br /></div>);
```

Comment:
what about `return null;` instead? There is no need to render something if nothing has to be rendered.